app/helpers/model_files_helper.rb

```diff
@@ -42,6 +45,8 @@ def slicer_icon_tag(slicer, alt:)
       "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons/webp/orcaslicer.webp"
     when :cura
       "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons/webp/cura.webp"
+    when :elegoo
+      "https://raw.githubusercontent.com/ELEGOO-3D/ElegooSlicer/refs/heads/main/resources/images/ElegooSlicer.svg"
```

Comment:
This is fine, but we'll also need to add `https://raw.githubusercontent.com` to the allowed CORS image sources at https://github.com/manyfold3d/manyfold/blob/b300969c1e99482d057f9cecdf6869d972c5dad2/app/controllers/application_controller.rb#L54